docs/resources/alert.md

```diff
@@ -1,5 +1,10 @@
+<!---
```

Comment:
ugh, this isn't supposed to end up here. I need to figure out a way to suppress this when these files are auto-generated from the .md.tmpl files. Maybe I'll just remove this whole block for now to avoid confusing people even more.